ft_databrowser.m

```diff
@@ -1806,9 +1806,9 @@ function redraw_cb(h, eventdata)
       
       % compute the time of the event
       eventtim(ievent) = (event(ievent).sample-begsample)/opt.fsample + opt.hlim(1);
-      
-      lh = ft_plot_line([eventtim(ievent) eventtim(ievent)], [-1 1], 'tag', 'event', 'color', eventcol{ievent}, 'hpos', opt.hpos, 'vpos', opt.vpos, 'width', opt.width, 'height', opt.height, 'hlim', opt.hlim, 'vlim', [-1 1]);
-      
+      % plot events as box or line (when duration = 0)
```

Comment:
I am afraid that in some cases event.duration would be [] (i.e. unknown) instead of 0. To make the code robust, I propose that you add something like

```
if isempty(event(ievent).duration) || event(ievent).duration==0
  plot line
else
  plot box  
end
```

Reply:
also, lines will be faster than boxes (which are 4 lines) in the cases when the duration is 0 or []

Reply:
Okay, I'll change that in a new commit!